src/scancode/help.py

```diff
@@ -24,6 +24,11 @@
 
     scancode -lc --package --ignore README --processes 4 --json-pp - samples/
 
+Scan a directory while ignoring header(.h) files . Scan for copyright and license.
+Print scan results to file as pretty formatted JSON.
```

Comment:
You use a redirect from the standard output to a file which is a more advanced feature. Could your example use a file directly?

Reply:
> Thank you! See my small comment

Ok Will do that Sir.